aws_cloudtrail_rules/aws_console_login_failed.yml

```diff
@@ -1,9 +1,9 @@
 AnalysisType: rule
 Filename: aws_console_login_failed.py
-RuleID: AWS.Console.LoginFailed
+RuleID: DEPRECATED RULE - AWS.Console.LoginFailed
```

Comment:
We should likely remove these from the repo after a single release

Reply:
Curious about your thoughts @nhakmiller

Reply:
yeah that would be a good workflow. The only reason we disable them is because before packs we couldn't remotely delete them. If the idea is customers should be staying up to date with the latest changes in this repo then removing after one release makes sense.